packages/disploy/src/structs/Modal.ts

```diff
@@ -0,0 +1,33 @@
+import type { App } from "../client";
+import { Base } from "./Base";
+import type { ActionRow } from "./ActionRow";
+
+export class Modal extends Base {
```

Comment:
Shouldn't be extending Base since it requires you to initiate with the App, which gives pretty bad DX.

```ts
import type { Command } from "disploy";

export default {
	name: 'modal',
	description: 'test modals!',

	async run(interaction) {
		const modal = new Modal(interaction.app);
		modal.set...
		interaction.showModal(modal);
	},
} satisfies Command;
```

Reply:
Resolved in 85a2bafe346c977558eefdc89bbbbb2f8f77af6c